controller/aks-cluster-config-handler.go

```diff
@@ -870,3 +886,38 @@ func (h *Handler) updateUpstreamClusterState(ctx context.Context, secretsCache w
 	logrus.Infof("Configuration for cluster [%s] was verified", config.Spec.ClusterName)
 	return config, err
 }
+
+// GetDefaultUpstreamTags gets the tags returned from the response returned by an update tags request sent to the Azure
+// API. Tags are returned in a TagsObject struct.
+func GetDefaultUpstreamTags(response containerservice.ManagedClustersUpdateTagsFuture) (containerservice.TagsObject, error) {
```

Comment:
The word `Default` here is not correct. This is just getting the tags on the cluster after the request. There doesn't seem to be any concept of "default" in that scenario.